src/engine/strat_engine/filesystem.rs

```diff
@@ -64,6 +97,33 @@ impl StratFilesystem {
         self.thin_dev.id()
     }
 
+    /// Return an target size for the filesystem over the thin device.
```

Comment:
ok - changed to : 

"Return an extend size for the thindev under the filesystem"